docs/book/src/introduction.md

```diff
@@ -0,0 +1,50 @@
+# The Cluster API Book
+
+**Note:** Impatient readers may head straight to [Quick
+Start](quick-start.md).
+
+**Using Cluster API v1alpha1? Check the [legacy
+documentation](https://release-0-1.cluster-api.sigs.k8s.io)**
+
+# What is the Cluster API?
+
+![](images/introduction.png)
+
+The Cluster API is a Kubernetes project to bring declarative, Kubernetes-style APIs to cluster creation,
```

Comment:
Can we embed this from https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/scope-and-objectives.md instead? It seems odd to duplicate this content in two different places.

Reply:
Yes! Anchors work everywhere :), just did it for both these goals and the README part